### PR TITLE
Migration from v08 to v1 arcticle

### PR DESCRIPTION
**Pull request by vladiulianbogdan**

### CODE REVIEW

**Comment by costinsin** [comment]
What about creating a new category instead of placing this in `Tutorials`? There should not be so many entries in the tutorials subsection.

**Comment by vladiulianbogdan** [comment]
> What about creating a new category instead of placing this in `Tutorials`? There should not be so many entries in the tutorials subsection.

Yea, we can move it whenever.

File: docs/tutorials/migration-to-v1.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Let's say you have a project named `my-project` in region `us-east-1`.
+
+When using genezio CLI version v0.8.0, the SDK will be named `@genezio-sdk/my-project_us-east-1`. Updating to genezio CLI v1.0.0 will change the SDK package name to `@genezio-sdk/my-project`.
+
+We advise adopting the updated SDK naming convention. To migrate you have to do the following steps:
+
+1. Make sure you don't have any reference in `scripts` in `package.json` to `@genezio-sdk/<project_name>_<project_region>`.
+
+```json
```
Comment (costinsin) [inline docs/tutorials/migration-to-v1.md:17]:
```suggestion
```json title="package.json"
```

File: docs/tutorials/migration-to-v1.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Guide on how to migrate to v1.0.0
+
+## genezio.yaml v2
+
+## Remove region from SDK
+
+In the genezio CLI v1.0.0 release, we updated the generated SDK npm package name by removing "region" for a more intuitive naming convention.
```
Comment (costinsin) [inline docs/tutorials/migration-to-v1.md:7]:
```suggestion
In the genezio CLI v1.0.0 release, we updated the generated SDK npm package name by removing `region` for a more intuitive naming convention. 
```

File: docs/tutorials/migration-to-v1.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  "name": "todo-list-ts",
+  "scripts": {
+    // diff-remove
+    "install-prod-sdk": "npm install @genezio-sdk/todo-list-ts_us-east-1@1.0.0-prod"
+    // diff-add
+    "install-prod-sdk": "npm install @genezio-sdk/todo-list-ts@1.0.0-prod"
+  },
+  "dependencies": {
+    // diff-remove
+    "@genezio-sdk/todo-list-ts_us-east-1": "npm:@genezio-sdk/todo-list-ts@^1.0.0-prod",
```
Comment (costinsin) [inline docs/tutorials/migration-to-v1.md:28]: I think the user will have something like:
```
"@genezio-sdk/todo-list-ts_us-east-1": "^1.0.0-prod",
```

File: docs/tutorials/migration-to-v1.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## genezio.yaml v2
+
+## Remove region from SDK
+
+In the genezio CLI v1.0.0 release, we updated the generated SDK npm package name by removing "region" for a more intuitive naming convention. 
+
+Let's say you have a project named `my-project` in region `us-east-1`.
+
+When using genezio CLI version v0.8.0, the SDK will be named `@genezio-sdk/my-project_us-east-1`. Updating to genezio CLI v1.0.0 will change the SDK package name to `@genezio-sdk/my-project`.
```
Comment (costinsin) [inline docs/tutorials/migration-to-v1.md:11]:
```suggestion
For instance, if you have a project called `my-project` in the `us-east-1` region, creating the SDK before Genezio CLI v1.0 will result in an NPM package called `@genezio-sdk/my-project_us-east-1`. From Genezio CLI v1.0 onward, the package will be named `@genezio-sdk/my-project`, maintaining the same functionality while being more intuitive.
```

Comment (costinsin) [reply docs/tutorials/migration-to-v1.md:11]: or the general way, choose the one that you like the most.

```suggestion
Creating the SDK before Genezio CLI v1.0 will result in an NPM package called `@genezio-sdk/<project_name>_<region>`. From Genezio CLI v1.0 onward, the package will be named `@genezio-sdk/<project-name>`, maintaining the same functionality while being more intuitive.
```

File: docs/tutorials/migration-to-v1.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+} from "@genezio-sdk/todo-list-ts_us-east-1";
+// diff-add
+} from "@genezio-sdk/todo-list-ts";
+import { useNavigate } from "react-router-dom";
+
+```
+
+4. Make sure that there are no more references to `@genezio-sdk/<project_name>_<project_region>`.
+
+It is not completely mandatory to switch right away. If you continue using the old name SDK package name, npm will create an alias in your `package.json`. In that case, your `package.json` will look like this:
```
Comment (costinsin) [inline docs/tutorials/migration-to-v1.md:53]:
```suggestion
It is not completely mandatory to switch right away. If you continue using the old name SDK package name, NPM will create an alias in your `package.json`. In that case, your `package.json` will look like this:
```

**Review by costinsin — APPROVED** [review]
LGTM

1. I would capitalize genezio -> Genezio
2. I would keep only the major and minor everywhere in the page. v1.0.0 -> v1.0, v0.8.0 -> v0.8. In my opinion it looks better.
3. I would capitalize npm -> NPM

**Pull request merged**